### ISSUE TITLE
Unable to start phantomjs with ghostdriver

### ISSUE THREAD

**Issue by randallm** [AlbanyCompSci/aeries-api#6]
I'm unable to execute Main.py, with the following error:

```
Traceback (most recent call last):
  File "Main.py", line 46, in <module>
    user_data = getUserData(login_data['email'], login_data['password'])
  File "Main.py", line 19, in getUserData
    session = AeriesSession.Session(email, password)
  File "/home/randall/workspace/aeries-api/AeriesSession.py", line 35, in __init__
    self.driver = webdriver.PhantomJS()
  File "/home/randall/workspace/aeries-api/local/lib/python2.7/site-packages/selenium/webdriver/phantomjs/webdriver.py", line 50, in __init__
    self.service.start()
  File "/home/randall/workspace/aeries-api/local/lib/python2.7/site-packages/selenium/webdriver/phantomjs/service.py", line 63, in start
    raise WebDriverException("Unable to start phantomjs with ghostdriver.", e)
selenium.common.exceptions.WebDriverException: Message: 'Unable to start phantomjs with ghostdriver.' ; Screenshot: available via screen 
```

I've installed the latest versions of bs4, python-dateutil, and selenium. Selenium standalone is launched with `java -jar selenium-server-standalone-2.37.0.jar -role hub`, and phantomjs is launched with `./phantomjs --webdriver=8080 --webdriver-selenium-grid-hub=http://127.0.0.1:4444`. How does my setup differ from yours?

**Comment by DavidHarrison** [comment]
To use PhantomJS with Selenium, I had to install local/phantomjs from the Arch Repos. I do not think I had to install anything else, though with Chrome, I had to install local/chromedriver.

**Comment by randallm** [comment]
How are you launching all the different clients? Are they running as daemons?

**Comment by DavidHarrison** [comment]
As a far as I am away, the WebDriver instance (which is preserved in the AeriesSession class) is the only initialization needed. I do not remember having to load any daemons.

**Comment by DavidHarrison** [comment]
On Debian, try http://packages.debian.org/unstable/web/phantomjs, or for source, try https://github.com/ariya/phantomjs.

**Comment by DavidHarrison** [comment]
I think I have figured it out now. At least on Ubuntu, the version of PhantomJS is 1.4, while on Arch it is 1.9. It was only with version 1.8 that GhostDriver was included with PhantomJS. GhostDriver a necessary part of the Selenium driver setup. To solve this on Ubuntu, I downloaded the binary from phantomjs.org/download.html, and copied bin/phantomjs to /usr/bin. This allowed the program to run.
